_content/home-page.md

```diff
@@ -2,10 +2,7 @@
 last_updated_time: 2021-08-09T02:58:39.509Z
 home_banner_image: https://res.cloudinary.com/wargabantuwarga/image/upload/v1627049958/hero_banner_desktop_zat71c.png
 ---
-#### Mau Konsultasi Medis dengan Dokter (Telemedicine)?
```

Comment:
You don't need to remove this part via codebase.
We can remove later from CMS.
Just focus on the new section.

Reply:
noted. just pushed the restored version from this file